### DISCUSSION TITLE
New translation in Slovak

### DISCUSSION THREAD

**Discussion by HDTS1** [sircharlo/meeting-media-manager#1027] [Translations]
I would like to help to translate M³ into a language I speak,Slovak (V/sk).

**Comment by mtdvlpr** [comment]
@HDTS1, thank you so much!

To get started, please [sign into GitLocalize](https://gitlocalize.com/) using your GitHub account, and then let me know when that's done by adding a comment in this chat. 

As language moderator, you will get notified of changes to the English original strings, have a nice and easy to use interface to review existing as well as untranslated strings in your language, and submit new translations to the repo easily.

**Reply by madroots** [reply]
Hello, I would like to translate from another account. From the one I am writing this. I am logged in gitlocalize.

**Reply by mtdvlpr** [reply]
@madroots, I have added you to Gitlocalize. There are 2 repos. The first one is for  M³ itself (the one from sircharlo). The other one (mtdvlpr), is for the documentation found at https://sircharlo.github.io/meeting-media-manager/.

**Reply by madroots** [reply]
Thank you very much for quick reply and letting me translate! 

May I ask, is there a way to use my new localization locally to understand context of the words? Sometimes it might really help. 
Its true I didn't yet read much of a manual yet, so feel free to point me there if thats already answered. 

>Thank you!

**Reply by mtdvlpr** [reply]
Yes, [CONTRIBUTING.md](https://github.com/sircharlo/meeting-media-manager/blob/master/CONTRIBUTING.md) explains how to run the application locally. To add your language to the selector, add it to the LOCALES array in `/src/renderer/constants/langs.json`. Then, have your translation strings in `/src/renderer/locales/sk.json`.

**Reply by madroots** [reply]
Thank you, have a happy life.

**Reply by mtdvlpr** [reply]
No worries, feel free to ask questions when you run into anything!

**Comment by mtdvlpr** [comment]
@madroots, if you want to test out your translations, [here](https://drive.google.com/drive/folders/1_2CzKieYGaK3LcYrEwzNQgfTpNReOSx9)'s an exe file for you to install :)

**Reply by madroots** [reply]
Thank you, I am on Linux but will carry this over to a Win PC and test it out later today. Thanks! :) Will report back

**Reply by mtdvlpr** [reply]
Nice, same. Just added an AppImage file to the same link :)

**Reply by madroots** [reply]
Thank you so much! I can test it out right away! Appreciate

**Reply by madroots** [reply]
I can already see lots of small problems. I start to fix it o.O

**Comment by mtdvlpr** [comment]
Dear @madroots, we have migrated to Crowdin for translations. Please see https://github.com/sircharlo/meeting-media-manager/discussions/722#discussioncomment-4633980 for details.

**Comment by madroots** [comment]
Hello, thank you very much. I noticed but haven't had a time yet. I will submit final changes once ready.

Thank you very much for your guidance.
